app/src/main/java/com/framgia/soundclound/data/model/Genres.java

```diff
@@ -0,0 +1,55 @@
+package com.framgia.soundclound.data.model;
+
+import android.databinding.BaseObservable;
```

Comment:
ERROR: Unused import: android.databinding.BaseObservable. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports